frontend/src/app/images/images.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="clr-row clr-justify-content-center">
   <div class="clr-col-4">
     <h1>Is this image a boy or a girl?</h1>
-    <h2 [hidden]="clickMessage == null">{{ clickMessage }}</h2>
+    <h2 [hidden]="shouldShowClickMessage">{{ clickMessage }}</h2>
```

Comment:
I first argued in my head about using a non-getter property and setting it manually every onClick (not seeing the alternative right in front of me). Good catch and far better solution.